.github/pull_request_template.md

```diff
@@ -1,9 +1,9 @@
 ### Description
 [Describe what this change achieves]
- 
+
```

Comment:
Sorry to reviewers for including these changes to remove the space on empty lines but it's been getting on my nerves for a while now and I don't want to waste the time making a new PR right now